course_catalog/models.py

```diff
@@ -18,3 +18,18 @@ class Course(models.Model):
     enrollment_end = models.DateTimeField(null=True)
     image = JSONField(null=True)
```

Comment:
Forgot about this one, maybe it makes sense to make this a URLField (`src` for the image) if there's only ever one, or if there's multiple then a `CourseImage` model.

Reply:
Either of those options seems fine with me.